tasks/system_packages.yml

```diff
@@ -5,15 +5,15 @@
 
 - name: ensure pre-defined default packages present
   yum:
-    name: "{{ item }}"
+    name: '{{ item }}'
```

Comment:
No, but since we prefer single-quotes in PHP/Python, I think we should do the same for Ansible.